### DISCUSSION TITLE
How to deploy my Next.js app to Netlify manually ?

### DISCUSSION THREAD

**Discussion by abedshaaban** [vercel/next.js#39424] [Help]
I Don't know how to deploy my app manually like drag and drop in netlify, when i drag and drop my .next file to netlify, i get a message( "Queueing build" ), i don't know what it mean and the site will not deploy. 
what should i do ?

**Comment by wwwdepot** [comment]
You should read Netlify docs https://docs.netlify.com/integrations/frameworks/next-js/
